src/connectors/rating-menu/connectRatingMenu.js

```diff
@@ -223,7 +223,7 @@ export default function connectRatingMenu(renderFn, unmountFn = noop) {
       _toggleRefinement(helper, facetValue) {
         const isRefined =
           this._getRefinedStar(helper.state) === Number(facetValue);
-        helper.clearRefinements(attribute);
+        helper.removeDisjunctiveFacetRefinement(attribute);
```

Comment:
Note that there is still a clearRefinement hanging out in the getWidgetSearchParameters which is still executed at this point.

Reply:
the one in gwsp is fine, because the refinement gets reapplied right after